tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -666,7 +673,9 @@ public function checkItemValues($contribution) {
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
-  public function testAssignProportionalLineItems() {
+  public function testAssignProportionalLineItems(): void {
+    // This test doesn't seem to manage financials properly, possibly by design
+    $this->isValidateFinancialsOnPostAssert = FALSE;
```

Comment:
Does this get reset for the tests that come after it?

Reply:
@seamuslee001 yep - I just put in a break point in the next test &

![image](https://user-images.githubusercontent.com/336308/127083365-90a699ed-cdc1-4871-b20d-c4d87e40680c.png)